jekyll/_cci2/collect-test-data.adoc

```diff
@@ -95,7 +95,7 @@ For information on viewing your storage usage, and calculating your monthly stor
 [#test-insights]
 == Test Insights
 
-NOTE: The test Insights feature is not currently supported for GitLab, GitHub App or Bitbucket Data Center projects. To find out if you authorized through the GitHub OAuth app or the CircleCI GitHub App, see the xref:github-apps-integration#[GitHub App integration] page.
+include::../_includes/partials/notes/standalone-unsupported.adoc[This feature is not supported for GitLab, GitHub App or Bitbucket Data Center]
```

Comment:
I don't know if we need to adjust this now, but I think talking about "projects" here is a good simplification but is not exact - I think it's the org type that determines if you have access or not.
+ hybrid orgs (OAuth + App) will have access to insights. I wonder if this description risks being confusing for some

Reply:
I'm updating note to be org specific not project specific